src/ui/components/VPNControllerView.qml

```diff
@@ -551,6 +551,8 @@ Rectangle {
         lineHeight: Theme.controllerInterLineHeight
         width: box.width - Theme.windowMargin * 3
         Accessible.ignored: connectionInfoVisible
+        Accessible.name: logoSubtitle.text
```

Comment:
I don't think `Accessible.name` and `Accessible.role` is necessary if we use the text from the subtitle as a description for the title. I would only repalce:
```
-        Accessible.ignored: connectionInfoVisible
+        Accessible.ignored: true
```
and lets add `Accessible.description: logoSubtitle.text`  around L542